gencapdefs.py

```diff
@@ -159,6 +159,12 @@
         url="https://wiki.znc.in/Query_buffers",
         standard="ZNC vendor",
     ),
+    CapDef(
+        identifier="EventPlayback",
+        name="draft/event-playback",
+        url="https://github.com/ircv3/ircv3-specifications/pull/362",
+        standard="Draft IRCv3",
```

Comment:
This should be `"proposed IRCv3"` instead of draft, as it hasn't been accepted and merged in.